src/adapters/defi.ts

```diff
@@ -372,13 +372,22 @@ class Defi extends Fetcher {
       amountArg,
     } = this.parseTipParameters(args)
     recipients = await this.parseRecipients(msg, targets, sender)
+    console.log(recipients, targets)
 
+    // check if recipient is author
+    if (targets.includes(`<@${authorId}>`)) {
```

Comment:
a có đọc cái request hồi Minh làm, nếu có author thì tự động bỏ qua mà nhỉ? còn khi nào tự tip author thôi thì mới error cannot tip themselves mà phải không?

Reply:
i fixed it